### PR TITLE
video capture

### PR DESCRIPTION
**Pull request by kevodwyer**
paired with video-capture branch in peergos repo

### CODE REVIEW

**Comment by kevodwyer** [comment]
Currently only available in Safari Tech Preview. Not yet available in iOS.  https://caniuse.com/#search=MediaRecorder

File: src/components/filesystem/index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = {
             normalSortOrder: true,
             clipboard:{},
             selectedFiles:[],
+            captureChunks:[],
```
Comment (ianopolous) [inline src/components/filesystem/index.js]: how big can this get?

File: src/components/video-capture/video-capture.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<transition name="modal">
+<div class="modal-mask" @click="close">
+    <div style="height:30%"></div>
```
Comment (ianopolous) [inline src/components/video-capture/video-capture.html]: would this be better as fullscreen? especially on mobile?

File: src/components/video-capture/index.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
+module.exports = {
+    template: require('video-capture.html'),
+    data: function() {
+        return {
+            buttonLabel: "start capture",
+            captureFilename: null,
+            timesliceInMs: 5000,
+            fps: 3,
```
Comment (ianopolous) [inline src/components/video-capture/index.js]: this sounds like a very low framerate?

File: src/components/filesystem/index.js (hunk 1 above)
Comment (kevodwyer) [reply src/components/filesystem/index.js]: each chunk is 5 seconds worth of data. The array decreases in size after each chunk is written to storage.

File: src/components/video-capture/video-capture.html (hunk 2 above)
Comment (kevodwyer) [reply src/components/video-capture/video-capture.html]: Yep, it needs tidy up. Something in the CSS means the view is not sized correctly.

File: src/components/video-capture/index.js (hunk 3 above)
Comment (kevodwyer) [reply src/components/video-capture/index.js]: It seems to be ignored anyway. I think this PR should be left in stasis until MacOS implementation is available.

**Pull request closed without merging**